### PR TITLE
Updated instructions on blower sub-assembly

### PR DESCRIPTION
**Pull request by martukas**
This includes some components designed and contributed by Foundry-M.
Assembly instructions for integration of blower into enclosed assembly have been written and illustrated.
Some less generally applicable items have been moved to pizza build BOM.

Purchasing supplier glossaries have been consolidated into one that is now in the "purchasing purchasing guidelines" page, with a brief asterisk link from each BOM.

Some renumbering of BOM items has also been done in multiple pages.

This is still a tentative page. Further changes are pending by @tijanim who will migrate tubing adapter and mounting plate drawings into Autodesk. It is also likely that the gaskets are not quite ideal. This further work will be in the scope of ticket #924 

Supplier glossary moves out of BOMs and into a dedicated purchasing guide page. Some price total estimates added to BOMs.

### CODE REVIEW

**Review by inceptionev — CHANGES_REQUESTED** [review]
manufacturing/brain/readme.md supplier links aren't rendering correctly in the rich text view.  (it shows the brackets instead).  Looks like the link targets haven't been changed to the alphanumeric designations.

File: manufacturing/brain/README.md [hunk 1]
```diff
@@ -8,49 +8,41 @@ For an upgrade, check the [Pizza build](../pizza_build).
...
+Additional notes:
+- Take note of the specifically RPI3+ or RPI4 components. Don't get the wrong ones.
+- If you are not (yet) able to get a PCB, get **A18 & A19**
+- If you are getting a PCB, and a power cable is included, you will not need **A15-17**, nor **B2**
+- If you are feeling stingy, don't get the touchscreen (**A3**). You can connect the RasPi to any monitor and use
+a mouse and keyboard to control it.
 
-| Item | Qty | Manufacturer | Part Number        | Price (USD) | Sources                   | Notes |
-| ---- |----:| -------------| ------------------ | --------:|-------------------------- | ----- |
-| 1    |   1 | ST Micro     | NUCLEO-L452-RE     |    14.43 | [M][1mouser] [R][1rs]     | STM32 Nucleo dev board, make sure to **NOT get the -P** version |
```
[the remark this answers is not in the document]
Comment (martukas) [reply manufacturing/brain/README.md:35]: Good catch!

[the remark this answers is not in the document]
Comment (inceptionev) [reply manufacturing/brain/README.md:35]: looks good

**Review by inceptionev — APPROVED** [review]
changes look good, approved!

**Pull request merged**